src/main/io/gps.c

```diff
@@ -241,7 +270,6 @@ typedef enum {
 
 gpsData_t gpsData;
 
-
 PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 0);
```

Comment:
Sorry I forgot but we should update new version here:
```suggestion
PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 1);
```